dotnet/interactive
# Script to make nuget packages for SqlTools ServiceLayer

This adds a script and a set of projects that produce a set of nuget packages containing the platform specific. SQL kernel implementations.

run the command:
    buildSqlTools.cmd
and wait.

To test it:
#i "nuget:C:\kevinransom\interactive\artifacts\packages\release\Shipping"
#r "nuget:Microsoft.SqlTools.ServiceLayer,3.0.0-release.49"

Note that your nuget cache will now contain a bunch od ServiceLayer packages.

I will work with Jon tomorrow, to figure out how to modify the sql kernel to make use of these packages.


In order to get package signing cibuild.ps1 has been updated to generate the packages.
